internal/statesync/stateprovider.go

```diff
@@ -369,10 +373,12 @@ func (s *stateProviderP2P) consensusParams(ctx context.Context, height int64) (t
 		}
 		iterCount++
 
+		// jitter+backoff the retry loop
+		timer.Reset(time.Duration(iterCount)*consensusParamsResponseTimeout + time.Duration(1+rand.Int63n(int64(iterCount)))) // nolint:gosec
```

Comment:
It might also be helpful to add an info log entry here so node operators can see where in the statesync process the node is at.

Reply:
logging here would log pretty frequently, and this is just to get the consensus params, so that seems a bit much.